### PR TITLE
improve backoff's code readability

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java [hunk 1]
```diff
@@ -31,26 +31,22 @@
...
-  int splitRegionBackoff = 20000;
-  int commitMaxBackoff = 3000;
-
-  /** 20 seconds */
-  int batchPrewriteBackoff = 20000;
-
-  /** 3 seconds */
-  int batchCommitBackoff = 3000;
+  int seconds = 1000;
+  int copBuildTaskMaxBackoff = 5 * seconds;
```
Comment (marsishandsome) [inline tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java]: it's better to use `final static int COP_BUILD_TASK_MAX_BACK_OFF = ...`

Comment (zhexuany) [reply tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java]: `final static` is useless in an interface type.

Comment (marsishandsome) [reply tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java]: u r right.

**Comment by marsishandsome** [comment]
LGTM

**Pull request merged**